src/index.js

```diff
@@ -1,16 +1,27 @@
 import createTest, { isTestLibrarySupported } from './create-test';
 
+const _returnChildState = (stateKey, stateObj) => {
```

Comment:
I'd prefer not to use `_` in front of function. Also no need for this to be an arrow function. And can we call this `getNestedState`